react/FilePicker/FilePickerBodyItem.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import { models } from 'cozy-client'
+import ListItem from '../MuiCozyTheme/ListItem'
+import ListItemIcon from '../MuiCozyTheme/ListItemIcon'
+import ListItemText from '../ListItemText'
+import Divider from '../MuiCozyTheme/Divider'
+import Icon from '../Icon'
+import FileTypeText from '../Icons/FileTypeText'
+import FileTypeFolder from '../Icons/FileTypeFolder'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+
+const {
+  file: { isDirectory, isFile }
+} = models
+
+const FilePickerBodyItem = ({
+  file,
+  isAccept,
+  multiple,
+  handleChoiceClick,
+  handleListItemClick,
+  onChangeRadio,
+  filesIdsSelected,
+  hasDivider
+}) => {
+  const hasFileChoice = isAccept.file && isFile(file)
+  const hasFolderChoice = isAccept.folder && isDirectory(file)
```

Comment:
My understanding of the following lines is that you always check if `hasFileChoice` or `hasFolderChoice` is `true` or if **none** of them is `true`

So I suppose you can merge those into one const named `hasChoice`? Or am I missing something?